docs/sources/installation/debian.md

```diff
@@ -49,12 +51,30 @@ Add this repository if you want beta releases:
 sudo add-apt-repository "deb https://packages.grafana.com/oss/deb beta main"
 ```
 
+#### Setup repo manually
+
+Some users have had problems with using `add-apt-repository` on ARM, thankfully it's trivial to setup the repo without it.
+
+Create `/etc/apt/sources.list.d/grafana.list` with the following content for stable releases:
+
+```bash
+deb https://packages.grafana.com/oss/deb stable main
+```
+
+Create `/etc/apt/sources.list.d/grafana.list` with the following content for beta releases:
+
+```bash
+deb https://packages.grafana.com/oss/deb beta main
+```
+
 Add our GPG key to install signed packages:
```

Comment:
Now this looks as if it's this is part of the "Setup repo manually" section, I guess that's unintentional? Should we make another subsection here?

Reply:
If you mean the gpg key setup, that's moved in master, will merge that into this pr.

Reply:
Does it look better now?